vividus/src/main/resources/properties/defaults/default.properties

```diff
@@ -76,6 +76,7 @@ system.allure.results.directory=${output.directory}/allure-results
 start-context-listener.cleanable-directories=${output.report-directory}
 
 known-issue-provider.fileName=known-issues.json
+known-issue-checker.allow-potentially-known-issues=true
```

Comment:
for users `known-issue-provider` or `known-issue-checker` don't have any difference, I propose to keep naming like `known-issue-provider...` (even if it belongs to another bean internally)